oak_remote_attestation/proto/v1/messages.proto

```diff
@@ -88,25 +88,15 @@ message GetPublicKeyResponse {
 }
 
 message InvokeRequest {
-  // Body of the request, encrypted with the enclave public key.
-  // TODO(#3442): Specify encryption format.
-  // TODO(#3442): Also pass client public or symmetric key to use for response encryption. Note that
-  // needs to be at least authenticated by the client, if not encrypted within this field.
-  // TODO(#3843): Use explicit `oak_crypto` messages.
-  bytes encrypted_body = 1;
   // Body of the request, encrypted using Hybrid Public Key Encryption (HPKE).
   // <https://www.rfc-editor.org/rfc/rfc9180.html>
-  oak.crypto.v1.EncryptedRequest encrypted_request = 2;
+  oak.crypto.v1.EncryptedRequest encrypted_request = 1;
```

Comment:
FWIW I think it's fine to leave the numbers as they are. Otherwise this is now a breaking change again, right?

Reply:
Will we be able to change the numbers later?

Reply:
If / when you change the numbers, it will be a breaking change.

Reply:
We can change the numbers internally first (and pass all of the internal tests). And then change externally.